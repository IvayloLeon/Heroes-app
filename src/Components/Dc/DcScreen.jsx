import React from 'react';
import { HeroList } from '../Heroes/HeroList'

export const DcScreen = ({publisher}) => {

    return (
        <div>
            <h1>DcScreen</h1>
            <hr />

            <HeroList publisher='DC Comics' />

        </div>
    )
}
