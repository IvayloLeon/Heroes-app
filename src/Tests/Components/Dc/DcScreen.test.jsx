import React from 'react'
import { DcScreen } from '../../../Components/Dc/DcScreen';
import {shallow} from 'enzyme';
import '@testing-library/jest-dom';

test('should ', () => {
    const wrapper = shallow(<DcScreen />);

    expect(wrapper).toMatchSnapshot();
});