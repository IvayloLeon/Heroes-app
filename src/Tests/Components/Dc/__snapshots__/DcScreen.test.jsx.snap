// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should  1`] = `
<div>
  <h1>
    DcScreen
  </h1>
  <hr />
  <HeroList
    publisher="DC Comics"
  />
</div>
`;
